gguf-py/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "gguf"
-version = "0.5.3"
+version = "0.6.0"
```

Comment:
We do want to bump it (so people who update the package get the new version). If it was me, I would just bump it to `0.5.4` though.

I am not 100% sure that's correct so it would absolutely be reasonable to get a second opinion from someone like cebtenzzre who could give you a definitive answer.

Reply:
I was going by the definition of [semantic versioning](https://semver.org/), where the version numbers are given by `MAJOR.MINOR.PATCH`. Since this is not a bug fix, bumping the minor version seemed the right thing to do here.

Reply:
You definitely may be right. In the past though, we've just bumped the patch version for small changes like adding a metadata field. I don't think the version is a big deal, so it's probably okay either way. I'd suggest asking someone that can give a definitive answer though.

Reply:
slaren is right, we should bump the minor version when we add new features, even small ones.